x-pack/plugin/otel-data/src/main/resources/component-templates/semconv-attribute-to-ecs@mappings.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
We had `semconv-resource-to-ecs@mappings.yaml`, but that maps resource attributes (and not attributes).

I think it'd make sense to have separate files for resource attributes vs attributes, so I introduced this new file.

Reply:
Since these aliases are only really relevant for logs, I'm wondering if we should just put this in `logs-otel@mappings`.

Reply:
If we have later other attributes that we want to map (I think we can safely assume it'll be the case), would that setup still work? I always forget the priorities... so won't one remove the others?

That's my only concern - if that's not an issue, then I agree, we could move those to `logs-otel@mappings` so this only applies to logs and not affect metrics and traces.

Reply:
Discussed via zoom: for now we move things to `logs-otel@mappings`.